src/w2form.js

```diff
@@ -1254,9 +1254,7 @@
             var url = (typeof this.url != 'object' ? this.url : this.url.get);
             if (url && this.recid != 0 && this.recid != null) {
                 this.request();
-            } else {
-                this.refresh();
-            }
+            } 
```

Comment:
because at the beginning  of .render( )  `var obj = this;` and at the end (line 1265) `setTimeout(function () { obj.resize(); obj.refresh(); }, 150); // need timer because resize is on timer` - i.e. `.refresh()` is executed anyway.
So, there isn't any problem (no?).